azure-cron.yml

```diff
@@ -118,3 +118,37 @@ jobs:
                --data "{\"text\":\"<!here> *FAILED* Daily Docs: <https://dev.azure.com/digitalasset/daml/_build/results?buildId=$(Build.BuildId)|$MESSAGE>\n\"}" \
                $(Slack.URL)
         condition: and(failed(), eq(variables['Build.SourceBranchName'], 'master'))
+
+  - job: docker_image
+    timeoutInMinutes: 20
+    pool:
+      name: 'linux-pool'
+    steps:
+      - checkout: self
+      - bash: |
+          set -euo pipefail
+
+          eval "$(dev-env/bin/dade-assist)"
+          echo $DOCKER_PASSWORD | docker login --username $DOCKER_LOGIN --password-stdin
+          RELEASES=$(curl https://api.github.com/repos/digital-asset/daml/releases -s | jq -r '. | map(select(.prerelease == false)) | map(.tag_name)[]')
+          DIR=$(pwd)
+          for version in $(echo $RELEASES | sed -e 's/ /\n/g'); do
+            EXISTS=$(curl 'https://hub.docker.com/v2/repositories/digitalasset/daml-sdk/tags/?page_size=10000' -s | jq -r '[.results[].name == "${version#v}"] | any')
```

Comment:
Should the curl not be outside the loop

Reply:
Right now the entire line is version-specific, but yes, the `curl` bit could be lifted out of the loop. Will do, thanks.